debian/changelog

```diff
@@ -1,3 +1,9 @@
+ola (0.10.7-1) unstable; urgency=low
+
+  * New upstream release
+
+ -- Simon Newton <nomis52@gmail.com>  Tue, 10 Jul 2018 02:32:00 +0000
```

Comment:
Best change this.

Reply:
Same as https://github.com/OpenLightingProject/ola/pull/1361#discussion_r160041808 :stuck_out_tongue:

Reply:
Please change this.

Reply:
Done